.gitignore

```diff
@@ -8,3 +8,5 @@ GPATH
 GRTAGS
 GTAGS
 stats
+.mypy_cache/
+.vscode/
```

Comment:
Please don't add these here. You can add your local gitignore under `.git/info/exclude` instead.